nixos/modules/services/misc/conduit.nix

```diff
@@ -0,0 +1,118 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.conduit;
+
+  format = pkgs.formats.toml {};
+  namespacedSettings.global = cfg.settings;
```

Comment:
Anybody have an idea how to make this more elegant?

Reply:
Let's just move global below the settings option and not do this dance. It's upstreams decision how they want their config nested.